src/main/java/com/cloudbees/plugins/credentials/CredentialsProvider.java

```diff
@@ -1515,20 +1515,37 @@ public static Fingerprint getOrCreateFingerprintOf(@NonNull Credentials c) throw
                 try {
                     Fingerprint fingerprint = getOrCreateFingerprintOf(c);
                     BulkChange change = new BulkChange(fingerprint);
+
+                    // Create a list of all current node names, the
+                    // credential will only be fingerprinted if it is one of these
+                    Set<String> jenkinsNodeNames = new HashSet<String>();
+                    for (Node n: Jenkins.getInstance().getNodes()) {
```

Comment:
I think the CI failure is because `Jenkins#getInstance` is [nullable](https://github.com/jenkinsci/jenkins/blob/2223dc7b547b5960d323ccc87a5cd62773dee988/core/src/main/java/jenkins/model/Jenkins.java#L711) in the version of Jenkins that the plugin is built against (1.625.3), so FindBugs doesn't like the chained `Jenkins.getInstance().getNodes()` call.

If this code is never intended to run while Jenkins is starting or shutting down then you could change it to `Jenkins.getActiveInstance` which will throw `IllegalStateException` if the instance is null, otherwise you could null check the result of `Jenkins.getInstance()` manually and do whatever makes sense in that case.

Reply:
See https://github.com/fraz3alpha/credentials-plugin/pull/1.